website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -16,7 +16,8 @@ This guide walks through an example of building a simple memcached-operator usin
   - `example.com` is used as the registry Docker Hub namespace in these examples.
   Replace it with another value if using a different registry or namespace.
   - [Authentication and certificates][image-reg-config] if the registry is private or uses a custom CA.
-
+- `CGO_ENABLED` environment variable set to `0`
+    - Example: `export CGO_ENABLED=0`
```

Comment:
It usually is not required. 
The problem here is how are you configuring your local Golang ENV that does not is as described in its documents by default. See: https://go.dev/doc/install

In your local env you are using this option to enable the golang.
Here we are saying that you MUST have golang installed and that means when you run go version from where you will execute the commands with SDK that for example works and return the version used which also must be compatible

Reply:
IHMO if you wish you can add a question / answer in the FAQ section
However, then you would need to describe what are the circumstances where it is required.
Why your env requires it and etc

Reply:
I'm happy to move this to the FAQ section since it isn't usually required and may be an unusual issue.

From what I can tell it seems like the problem could be related to arch & distro combinations and whether or not the proper C libraries are installed with `gcc` and whether or not `gcc` is installed by default (ex: Fedora on s390x had to have `CGO_ENABLED=0` even after installing `gcc` but Ubuntu on s390x only needed `gcc` installed).

Thanks for bringing that up!

Reply:
Moved to FAQ in commit https://github.com/operator-framework/operator-sdk/pull/5650/commits/1b69eea7078390839aedc1b878565c0a96e3761d